stripe/src/main/java/com/stripe/android/StripePaymentController.kt

```diff
@@ -355,7 +358,14 @@ internal class StripePaymentController internal constructor(
                     authenticator.onAuthenticationRequest(nextActionData.mobileData.data)
                 return AlipayAuthResult(
                     when (output[RESULT_FIELD]) {
-                        RESULT_CODE_SUCCESS -> StripeIntentResult.Outcome.SUCCEEDED
+                        RESULT_CODE_SUCCESS -> {
+                            nextActionData.mobileData.authCompleteUrl?.let {
+                                runCatching {
```

Comment:
should we move `runCatching` into the implementation of `retrieveObject`?

Reply:
Sure. Depends on how else this going to be used (if at all) but I suppose we can just figure that out if and when it ever happens.

Reply:
Actually, on second thought, it seems like all other methods in StripeApiRepository allow exceptions to bubble up. Don't feel particularly strongly but seems like we should be consistent here?